tools/internal_ci/linux/arm64/grpc_bazel_test_c_cpp_in_docker.sh

```diff
@@ -25,5 +25,13 @@ cd /var/local/git/grpc
 # tests require port server to be running
 python3 tools/run_tests/start_port_server.py
 
+python3 tools/run_tests/python_utils/bazel_report_helper.py --report_path bazel_test_c_cpp
+
 # test gRPC C/C++ with bazel
-tools/bazel test --config=opt --test_output=errors --test_tag_filters=-no_linux,-no_arm64 --build_tag_filters=-no_linux,-no_arm64 --flaky_test_attempts=1 --runs_per_test=1 //test/...
+bazel_test_c_cpp/bazel_wrapper \
+  --bazelrc=tools/remote_build/include/test_locally_with_resultstore_results.bazelrc \
+  test --config=opt \
```

Comment:
Are we sure we want to remove --flaky_test_attempts=1 and --runs_per_test=1?

Reply:
yes, those flags were initially used to assess the level of test flakiness on ARM64, but we're past that point and we no longer need them. I'm trying to make the arm64 config as close to the x64 config as possible.